kontroller/player.cpp

```diff
@@ -572,7 +572,7 @@ void Player::handlePlayerStatus_()
 						auto nosub = new Subtitle(this);
 						nosub->setIndex(-1);
 						nosub->setLanguage("");
-						nosub->setName(tr("No subtitle"));
+						nosub->setName(tr("No subtitles"));
```

Comment:
Almost sure this is correct.

Reply:
You probably know better than i do. Ok for me.